testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -719,11 +719,28 @@ def pillar_get(key, minion)
   step %(I enter "#{ENV['VIRTHOST_XEN_PASSWORD']}" as "password") if host == "xen_server"
 end
 
-And(/^I cleanup minion "([^"]*)"$/) do |minion|
+And(/^I clean up the minion's cache on "([^"]*)"$/) do |minion|
   raise "#{minion} is not a salt minion" unless minion.include? 'minion'
   node = get_target(minion)
-  node.run('systemctl stop salt-minion')
+  node.run_until_ok('systemctl stop salt-minion')
   node.run('rm -Rf /var/cache/salt/minion')
+  # TODO: Ideally we should remove /etc/salt/* /var/run/salt/* /var/log/salt/* and salt and salt-minion packages
+  #       But we can't do that as we don't have products synced, so it will fail installing salt and sal-minion
```

Comment:
"salt-minion"

In theory, we could re-enable original system repositories while we reinstall the packages... but... complicated, error-prone.

Reply:
I made a try about that in the past, and found some problems.
Anyhow, yeah, something to research. Better only clean the cache for now.